source/gui/__init__.py

```diff
@@ -924,3 +924,64 @@ def shouldConfigProfileTriggersBeSuspended():
 
 def _isDebug():
 	return config.conf["debugLog"]["gui"]
+
+class AskAllowUsageStatsDialog(wx.Dialog):
+	"""A dialog asking if the user whishes to allow NVDA usage stats to be collected by NV Access."""
+
+	def __init__(self, parent):
+		# Translators: The title of the dialog asking if usage data can be collected 
+		super(AskAllowUsageStatsDialog, self).__init__(parent, title=_("NVDA  Usage Data Collection"))
+		mainSizer = wx.BoxSizer(wx.VERTICAL)
+		sHelper = guiHelper.BoxSizerHelper(self, orientation=wx.VERTICAL)
+
+		# Translators: A message asking the user if they want to allow usage stats gathering
+		message=_("In order to improve NVDA in the future, NV Access wishes to collect usage data from running copies of NVDA. "
+			"Data includes Operating System version, NVDA version, language, country of origin, plus certain NVDA configuration such as current synthesizer, braille display and braille table. " 
+			"No spoken or braille content will be ever sent to NV Access.  Please refer to the User Guide for a current list of all data collected.\n"
+			"Do you wish to allow NV Access to periodically collect this data in order to improve NVDA?")
+		sHelper.addItem(wx.StaticText(self, label=message))

```

Comment:
Currently this text causes the dialog to expand to the full width of the screen, I would suggest using something like the following so that the lines are wrapped when they hit some max length:
```
        sText = sHelper.addItem(wx.StaticText(self, label=message))
        # the wx.Window must be constructed before we can get the handle.
        import windowUtils
        self.scaleFactor = windowUtils.getWindowScalingFactor(self.GetHandle())
        sText.Wrap(self.scaleFactor*600) # 600 was fairly arbitrarily chosen by a visual user to look acceptable on their machine.
```